firmware/thruna/BMS/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -7,11 +8,16 @@ void App_SetPeriodicCanSignals_Imd(struct Imd *imd)
     App_CanTx_BMS_ImdDutyCycle_Set(App_Imd_GetPwmDutyCycle(imd));
 
     const struct Imd_Condition condition = App_Imd_GetCondition(imd);
-    App_CanTx_BMS_ImdCondition_Set((uint8_t)condition.name);
+    App_CanTx_BMS_ImdCondition_Set((ImdCondition)condition.name);
```

Comment:
Maybe we should change the type of condition.name to ImdCondition so we don't need this cast? (and so there's only one enum as opposed to two currently)